openghg/retrieve/_access.py

```diff
@@ -481,3 +485,52 @@ def _scale_convert(data: Dataset, species: str, to_scale: str) -> Dataset:
     data.attrs["scale"] = to_scale
 
     return data
+
+
+multDataTypes = Union[List[ObsData], List[FootprintData], List[FluxData]]
+
+
+def metadata_difference(data: multDataTypes,
+                        params: Optional[list] = None,
+                        print_output: bool = True) -> list:
+    """
+    Check differences between metadata for returned data objects.
+
+    Args:
+        data : Multiple data objects e.g. multiple ObsData as a list
+        params : Specific metadata parameters to check. If None all parameters will be checked
+        print_output : Summarise and print output to screen.
+
+    Returns:
+        list : Keys from the metadata with differences
+
+        If print_output is True:
```

Comment:
I think this could be moved to the description section of the docstring instead of the Returns section.